test/DynamoCoreWpfTests/PreferencesViewModelTests.cs

```diff
@@ -157,10 +157,12 @@ public void PathManagerWithDifferentHostTest()
             string dynamoRevitUserDataDirectory = "C:\\Users\\user\\AppData\\Roaming\\Dynamo\\Dynamo Revit\\3.1";
             string dynamoRevitCommonDataDirectory = "C:\\ProgramData\\Autodesk\\RVT 2024\\Dynamo\\3.1";
             string dynamoRevitSamplesPath = "C:\\ProgramData\\Autodesk\\RVT 2024\\Dynamo\\samples\\en-US";
+            string dynamoRevitTemplatesPath = "C:\\ProgramData\\Autodesk\\RVT 2024\\Dynamo\\templates\\en-US";
```

Comment:
You could refactor this code with `Path.Combine` to use the common directory "C:\\ProgramData\\Autodesk\\RVT 2024\\Dynamo"
Something like:
string dynamoRevitCommonDataDirectory = Path.Combine(RVTForDynamoDir,"3.1") ;

Reply:
I can update the test